app/src/main/java/com/el/yello/presentation/onboarding/activity/OnBoardingActivity.kt

```diff
@@ -79,11 +79,13 @@ class OnBoardingActivity :
         val bundle: Bundle? = intent.extras
         intent.apply {
             if (bundle != null) {
-                viewModel.kakaoId = bundle.getLong(EXTRA_KAKAO_ID, 0).toString()
-                viewModel.nameText.value = bundle.getString(EXTRA_NAME, "")
-                viewModel.gender = bundle.getString(EXTRA_GENDER, "")
-                viewModel.email = bundle.getString(EXTRA_EMAIL, "")
-                viewModel.profileImg = bundle.getString(EXTRA_PROFILE_IMAGE, "")
+                with(viewModel) {
+                    kakaoId = bundle.getLong(EXTRA_KAKAO_ID, 0).toString()
+                    nameText.value = bundle.getString(EXTRA_NAME, "")
+                    gender = bundle.getString(EXTRA_GENDER, "")
+                    email = bundle.getString(EXTRA_EMAIL, "")
+                    profileImg = bundle.getString(EXTRA_PROFILE_IMAGE, "")
+                }
```

Comment:
```suggestion
                with(viewModel.bundle) {
                    kakaoId = getLong(EXTRA_KAKAO_ID, 0).toString()
                    nameText.value = getString(EXTRA_NAME, "")
                    gender = getString(EXTRA_GENDER, "")
                    email = getString(EXTRA_EMAIL, "")
                    profileImg = getString(EXTRA_PROFILE_IMAGE, "")
                }
```
요기두요!

Reply:
저렇게 한번에 묶지는 못하고 with로 한번더 묶어야할 것 같아요 ..!

```suggestion 
               with(viewModel) {
                    with(bundle) {
                        kakaoId = getLong(EXTRA_KAKAO_ID, 0).toString()
                        nameText.value = getString(EXTRA_NAME, "")
                        gender = getString(EXTRA_GENDER, "")
                        email = getString(EXTRA_EMAIL, "")
                        profileImg = getString(EXTRA_PROFILE_IMAGE, "")
                    }
                }

Reply:
헉 그러네 bundle이 뷰모델 변수가 아니었네요 😅 잘못봣습니다...

Reply:
두번 묶음 이상하네여 그대로 둡시다!!